cmd/reporting-operator/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/operate-first/curator-operator/internal"
+	"github.com/operate-first/curator-operator/pkg/operator"
+	"github.com/operate-first/curator-operator/pkg/signals"
+	"github.com/spf13/cobra"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+const (
+	defaultServerPort = ":8080"
+)
+
+var (
+	reportLog = ctrl.Log.WithName("report")
+)
+
+var (
+	cfg operator.Config
+
+	rootCmd = &cobra.Command{
+		Use:   "reporting-operator",
+		Short: "",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			return cmd.Help()
+		},
+	}
```

Comment:
I think we can consolidate the root and start cobra.Command variables into a single CLI command. We should also aim to avoid using global package variables as much as possible, so I added a review comment in https://github.com/operate-first/curator-operator/pull/9/files#r847717437 that creates a constructor function that's responsible for spitting out the cobra.Command that the main function can use.